plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -412,29 +412,8 @@ class CodeWhispererCodeScanManager(val project: Project) {
     fun getOverlappingScanNodes(file: VirtualFile, range: TextRange): List<DefaultMutableTreeNode> = synchronized(scanNodesLookup) {
         scanNodesLookup[file]?.mapNotNull { node ->
             val issue = node.userObject as CodeWhispererCodeScanIssue
-            if (issue.textRange?.overlaps(range) == true) node else null
-        } ?: listOf()
-    }
-
-    fun updateScanNodesForIssuesOutOfTextRange(fixedIssue: CodeWhispererCodeScanIssue) {
```

Comment:
This function updates issue scan nodes when the Applied Fix is out of issue text range(apply fix text change is not in yellow line area). Have you checked for this case

Reply:
Yes in this case we are now removing the issue directly by findingId